### ISSUE TITLE
[Bug Report] FileInput does not work in iOS

### ISSUE THREAD

**Issue by Darkside73** [vuetifyjs/vuetify#11502]
### Environment
**Vuetify Version:** 2.2.30
**Vue Version:** 2.6.11
**Browsers:** Google Chrome, Safari
**OS:** iOS

### Steps to reproduce
Open link and attach any file several times

### Expected Behavior
File attached every time

### Actual Behavior
File attached only once in 5-10 total attempts

### Reproduction Link
<a href="https://codepen.io/darkside73/full/ZEbNxYM" target="_blank">https://codepen.io/darkside73/full/ZEbNxYM</a>

### Other comments
onChange event does not fire as well when attach fails

**Comment by nekosaur** [comment]
duplicate of https://github.com/vuetifyjs/vuetify/issues/11455
